docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,6 @@
+##Contributing
```

Comment:
Let's surface up a bit to ./CONTRIBUTING.md so it's more obvious from the top-level... AND make sure we have a link to it from the top-level README.md

Reply:
ok